R/subset_draws.R

```diff
@@ -55,6 +61,22 @@ subset_draws.draws_matrix <- function(x, variable = NULL, iteration = NULL,
   iteration <- check_iteration_ids(iteration, x, unique = unique)
   chain <- check_chain_ids(chain, x, unique = unique)
   draw <- check_draw_ids(draw, x, unique = unique)
+
+  if (exclude) {
```

Comment:
please input check exclude via as_one_logical

Reply:
Thanks! Keep forgetting to do this. I now added input checking into the check_* functions, as is done with the the other arguments `regex` and `unique`. This does mean that the inputs are not checked if all `draw`, `iteration`, `variable`, `chain` are NULL though